src/resources/postcss/components/full/buttons/_solid.pcss

```diff
@@ -73,4 +73,21 @@
 			}
 		}
 	}
+
+	/* -------------------------------------------------------------------------
+	 * Theme Overrides - Twenty TwentyOne
+	 * ------------------------------------------------------------------------- */
+
+	 .tribe-theme-twentytwentyone & {
+
+		button::not(:hover):not(:active) {
+			background-color: inherit;
+			color: inherit;
+		}
```

Comment:
got 2 colons on the first `not()`, also this should go in `src/resources/postcss/base/full/_buttons.pcss`

Reply:
Thanks! added the changes.